src/libraries/System.Data.Common/src/System/Data/Common/DbException.cs

```diff
@@ -21,5 +21,6 @@ public abstract class DbException : System.Runtime.InteropServices.ExternalExcep
         protected DbException(System.Runtime.Serialization.SerializationInfo info, System.Runtime.Serialization.StreamingContext context) : base(info, context)
         {
         }
+        public virtual string? SqlState => null;
```

Comment:
Here's a suggestion:

```suggestion
        /// <summary>
        /// <para>
        /// For database providers which support it, contains a standard SQL 5-character return code indicating the success or failure of
        /// the database operation. The first 2 characters represent the <strong>class</strong> of the return code (e.g. error, success),
        /// while the last 3 characters represent the <strong>subclass</strong>, allowing detection of error scenarios in a
        /// database-portable way.
        /// </para>
        /// <para>
        /// For database providers which don't support it, or for inapplicable error scenarios, contains <see langword="null" />.
        /// </para>
        /// </summary>
        /// <returns>
        /// A standard SQL 5-character return code, or <see langword="null" />.
        /// </returns>
        public virtual string? SqlState => null;
```